utils/stats.py

```diff
@@ -18,6 +18,8 @@
 from utils.common import to_thread
 from utils.problems import fetch_problems_solved_and_rank
 
+from PIL import Image
```

Comment:
I'd recommend organising your imports, there is a command in vscode to do so in interested. Dw about it, I'll do that separately and also auto format the code to match the rest of the formatting, although it looks alright so far anyways